chemprop/cli/utils/utils.py

```diff
@@ -34,11 +34,11 @@ def _pop_attr_d(o: object, attr: str, default: Any | None = None) -> Any | None:
 
 
 def validate_loss_function(
-    predictor_ffn: Type[predictors._FFNPredictorBase], criterion: Type[loss.LossFunction]
+    predictor_ffn: Type[predictors._FFNPredictorBase], criterion: Type[loss.ChempropMetric]
 ):
     match predictor_ffn:
         case predictors.RegressionFFN:
-            if criterion not in (loss.MSELoss, loss.BoundedMSELoss):
+            if criterion not in (loss.MSE):
```

Comment:
If you still want this to be a tuple, it needs to be `(loss.MSE, )`

Reply:
This is removed after merging v2-1/dev into this branch.